client/src/component/signin/index.jsx

```diff
@@ -0,0 +1,128 @@
+import { React, useState } from 'react';
+import {
+  Layout,
+  Row,
+  Col,
+  Image,
+  Form,
+  Button,
+  Input,
+  message,
+} from 'antd';
+
+import axios from 'axios';
+import 'antd/dist/antd.css';
+import './index.css';
+
+const {
+  Content,
+} = Layout;
+
+function Signin() {
+  const [userData, setUserData] = useState({ email: '', password: '' });
+
+  const handleInputChange = (event) => {
+    const { name, value } = event.target;
+    setUserData({
+      ...userData,
+      [name]: value,
+    });
```

Comment:
Do you set the user data with the old one? I mean if there is already data in your state you merge it ?

Reply:
This handles both email and password input if i remove `...userData` it will only handle the password input

Reply:
Your right